src/scripts/generateBridgeList.ts

```diff
@@ -126,30 +126,42 @@ export async function generateBridgedList(params: GenerateBridgedListParams): Pr
 
 
   // Create a Map of symbols to the additional tokens to add
-  const tokenInfoBySymbol = tokensToAdd.reduce((acc, token) => {
+  const onchainInfoBySymbol = tokensToAdd.reduce((acc, token) => {
```

Comment:
Symbols are not unique. Why not use the address as key?

Reply:
I used symbols because, in case the CSV has a token with the same symbol I will override that address. It was the case for a few tokens where the bridged tokens where not good

Normally we prefer addresses, but in this case, our lists will not have duplicated symbols.

does it make sense?